src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -797,9 +804,10 @@ async function applyAutoInsertEdit(e: vscode.TextDocumentChangeEvent, token: vsc
     const request: RoslynProtocol.OnAutoInsertParams = {
         _vs_textDocument: textDocument,
         _vs_position: position,
-        _vs_ch: change.text,
+        _vs_ch: changeTrimmed,
```

Comment:
so we remove whitespace, but I think I still don't understand why we need to remove whitespace.  Does the summary text not get formatted properly if the line is already indented?

Reply:
Correct. The trigger characters looks for a new line character and the change.Text is something akin to "\n   ", which I presume is the spaces needed to bring the cursor to line up with the comment. So we need to remove whitespace to match the trigger character and, in the OnAutoInsertHandler, it again looks to see the changed text exactly matches the new line character.